recipe/meta.yaml

```diff
@@ -1,37 +1,61 @@
-{% set version = "0.7.0" %}
-{% set sha256 = "1aaadcd69338252ade4f7ec1265e1a19184bf916d84c9b7df095f423948cb89f" %}
+{% set version = "0.10.0" %}
 
 package:
   name: build
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/b/build/build-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  git_url: https://github.com/pypa/build
+  git_rev: {{ version }}
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   entry_points:
     - pyproject-build = build.__main__:entrypoint
 
 requirements:
+  build:
+    - git  # [not win]
   host:
-    - python >=3.6
+    - python
     - pip
+    - wheel
+    - flit-core 3.8.0
   run:
-    - python >=3.6
-    - packaging
-    - pep517 >=0.9.1
-    - tomli
-    - importlib-metadata
+    - python
+    - packaging >=19.0
+    - pyproject_hooks
+    - importlib-metadata >=0.22  # [py<38]
+    - tomli >=1.1.0  # [py<311]
+
+{% set tests_to_skip = "test_build_wheel or test_build_sdist" %}
```

Comment:
Why do we skip these tests?

Reply:
Forgot to add a couple of comment. Will add them.